chrome/locale/en-US/nostalgy.properties

```diff
@@ -0,0 +1,4 @@
+helloMessage=Hello World!
+helloMessageTitle=Hello
```

Comment:
What are these properties?

Reply:
nostalgy 0.2.34 has that as well, so it is old.